modules/common/DownloadResource.py

```diff
@@ -70,33 +70,15 @@ def execute_download_threaded(self, resource_info):
 
     def ftp_download(self, resource_info: Dict) -> str:
         print("Start to download\n\t{uri} ".format(uri=resource_info.uri))
-        filename = self.set_filename(resource_info.output_filename)
-        urllib.request.urlretrieve(resource_info.uri, filename)
-        urllib.request.urlcleanup()
-
-        return filename
-
-    def downloadHttp(self, resource_info, retry_count=3, proxy=None, data=None):
-        headers = {
-            "User-Agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.100 Safari/537.36"
-        }
-        destination_filename = self.set_filename(resource_info.output_filename)
-        if resource_info.uri is None:
-            return None
         try:
-            req = urllib.request.Request(resource_info.uri, headers=headers, data=data)
-            cookie = cookiejar.CookieJar()
-            cookie_process = urllib.request.HTTPCookieProcessor(cookie)
-            opener = urllib.request.build_opener()
-            if proxy:
-                proxies = {urllib.urlparse(resource_info.uri).scheme: proxy}
-                opener.add_handler(urllib.request.ProxyHandler(proxies))
-            content = opener.open(req).read()
-        except urllib.error.URLError as e:
-            print('downloadHttp download error:', e.reason)
-            content = None
-            if retry_count > 0:
-                if hasattr(e, 'code') and 500 <= e.code < 600:
-                    return self.download(resource_info.uri, retry_count - 1, headers, proxy, data)
+            filename = self.set_filename(resource_info.output_filename)
+            urllib.request.urlretrieve(resource_info.uri, filename)
+            urllib.request.urlcleanup()
+        except Exception:
+            logger.error("Warning: FTP! {file}".format(file=resource_info.uri))
+            # try with wget temp solution
+            cmd = 'curl ' + resource_info.uri + ' --output ' + filename
+            logger.info("wget attempt {cmd}".format(cmd=cmd))
+            subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
```

Comment:
Why do we have this fall-back option using `wget`? Did you find a problem with the `urllib` library?

Reply:
I added a crypto message #try with wget temp solution :)

Reply:
EBI FTP started to reply  ConnectionResetError: [Errno 104] Connection reset by peer.
From an exchange of email they suggested us to use wget...
What do you think @JarrodBaker ? Can we leave it and add #todo message or open a ticket about it?

Reply:
I'm happy to leave it in. Maybe just add a comment like you've done here so the next person isn't wondering too. :)

Reply:
Replace the comment with the explanation about the issue